### PR TITLE
add provider for static routes

### PR DESCRIPTION
**Pull request by krichprollsch**

### CODE REVIEW

File: src/SitemapGenerator/Provider/SimpleProvider.php [hunk 1]
```diff
@@ -0,0 +1,49 @@
+<?php
+
+namespace SitemapGenerator\Provider;
+
+use Symfony\Component\Routing\RouterInterface;
```
Comment (K-Phoen) [inline src/SitemapGenerator/Provider/SimpleProvider.php]: Not used.

File: tests/SitemapGenerator/Tests/Provider/SimpleProviderTest.php [hunk 2]
```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace SitemapGenerator\Tests\Provider;
+
+use \Mockery;
+
+use SitemapGenerator\Entity\Url;
+use SitemapGenerator\Provider\SimpleProvider;
+use SitemapGenerator\Tests\Fixtures\News;
```
Comment (K-Phoen) [inline tests/SitemapGenerator/Tests/Provider/SimpleProviderTest.php:7]: Same here, useless import (Mockery isn't used either)

File: tests/SitemapGenerator/Tests/Provider/SimpleProviderTest.php [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+
+/**
+ * @runTestsInSeparateProcesses
```
Comment (K-Phoen) [inline tests/SitemapGenerator/Tests/Provider/SimpleProviderTest.php]: Should not be necessary.

**Comment by K-Phoen** [comment]
There are a few little things that need to be fixed but the PR looks good, thanks!

**Comment by krichprollsch** [comment]
done.
Do you know why these errors were not shown into scrutinizer report ?

**Comment by K-Phoen** [comment]
The configuration used for the scrutinizer inspections is rather old and may be incomplete/incorrect. I'll look into it.

Thanks for your PR btw :)

**Pull request merged**